src/avfoundation.cs

```diff
@@ -1836,6 +1838,14 @@ interface AVAudioSessionSecondaryAudioHintEventArgs {
 
 	delegate void AVPermissionGranted (bool granted);
 
+	[Native]
+	public enum AVAudioSessionInterruptionReason : ulong
+	{
+		Default = 0,
+		AppWasSuspended = 1,
+		BuiltInMicMuted = 2
```

Comment:
```suggestion
		BuiltInMicMuted = 2,
```
Smaller diff next time.